src/screens/BookDetail/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Link, useParams } from 'react-router-dom';
+import { t } from 'i18next';
+
+import { getBookDetail } from '../../services/bookService';
+import useRequest from '../../app/hooks/useRequest';
+import { PATHS } from '../../constants/paths';
+import btnBack from '../../app/assets/btn-back.png';
+
+import styles from './styles.module.scss';
+
+function BookDetail() {
+  const { bookId } = useParams();
+  const [isLoading, submitError, response] = useRequest({
+    request: getBookDetail,
+    payload: bookId
+  });
+
+  const book = response && response.ok ? response.data : {};
+
+  return (
+    <>
+      {response && !response.ok && <p>{t('Errors:apiError')}</p>}
+      {submitError && <p>{submitError}</p>}
+      {isLoading && <p>{t('General:loadingContent')}...</p>}
+      {!isLoading && response && response.ok && (
+        <div className="column center">
+          <div className={clsx(styles.options, 'row', 'middle')}>
+            <img src={btnBack} />
+            <Link to={PATHS.root}>
+              <span className={styles.option}>{t('General:back')}</span>
+            </Link>
+          </div>
+          <div className={clsx(styles.book, 'row', 'space-around')}>
```

Comment:
use clsx only to validate properties 
```suggestion
          <div className={` ${styles.book} row space-around`}>
```

Reply:
I don't mind this